template.yml

```diff
@@ -827,6 +873,26 @@ Resources:
         MetricNamespace: !Sub "${AWS::StackName}-ZoomIngesterLogMetrics"
         MetricName: "MinutesInPipeline"
 
+  StartOperationMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    Properties:
+      LogGroupName: !Ref OpencastOpCountsLogGroup
+      FilterPattern: "{$.message.operation_counts.START_OPERATION > 0}"
```

Comment:
these `START_OPERATION` / `START_WORKFLOW` operations aren't really relevant to anything

Reply:
I need to think about what we'd actually want to track here (if anything). Let's just skip these for now.